src/simulators/superoperator/superoperator.hpp

```diff
@@ -53,7 +53,7 @@ class Superoperator : public DensityMatrix<data_t> {
   //-----------------------------------------------------------------------
   
   // Set the size of the vector in terms of qubit number
-  virtual void set_num_qubits(size_t num_qubits) override;
+  void set_num_qubits(size_t num_qubits);
```

Comment:
Yep, we are using this virtual method in the constructor, which is dangerous because of the way objects are being constructed at runtime, anyway we explicitly calling the parent method ins the implementation, so there's no need for it to be virtual.